plugins/source/github/resources/services/actions/workflowJobs.go

```diff
@@ -0,0 +1,51 @@
+package actions
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/google/go-github/v49/github"
+)
+
+func WorkflowJobs() *schema.Table {
```

Comment:
Can we make this function private by renaming it to `WorkflowJobs`? That would ensure this relational table is never used in the top level table list